### PR TITLE
Add full name for cantons

### PR DESCRIPTION
**Pull request by eduardocorrearaujo**

### CODE REVIEW

File: for_other_cantons.py [hunk 1]
```diff
@@ -294,28 +310,55 @@ def plot_forecast_canton(table_name,canton, curve, title= None):
...
     st.title('Number of cases and Hospitalizations')
     
-
-    fig_c, last_date, last_cases = plot_cases_canton(canton)
-
+    fig_c, last_date = plot_cases_canton(full_name_canton, canton)
     
     st.write(f'''
-             The graphs below show the number of cases and hospitalizations in {canton}
+             The graphs below show the number of cases and hospitalizations in Geneva
```
Comment (fccoelho) [inline for_other_cantons.py]: Here the string should be
```python
f"... hospitalizations in {full_name_canton}"
```

File: forecast.py [hunk 2]
```diff
@@ -388,7 +471,7 @@ def download_button(object_to_download, download_filename, button_text, pickle_i
 
 def app(): 
     
-    st.title('Switzerland COVID-19 Hospitalizatiokn forecasts')
```
Comment (fccoelho) [inline forecast.py]: Esta linha já está corrigida no ramo main: https://github.com/thegraphnetwork/COVID-CH-dashboard/blob/0b3acc9e4daec8b56d2007f2b02d0394b389a0b7/forecast.py#L391

File: get_data.py [hunk 3]
```diff
@@ -19,6 +19,15 @@
...
+    '''
+    Function to get the cases and hosp data in the database
+    '''
+    df = pd.read_sql_table(f'foph_{curve}', engine_public, schema = 'switzerland', index_col = 'datum', columns=[ 'geoRegion', 'entries'])
+    df.index = pd.to_datetime(df.index)
+    
+    return df
+
 
 def build_lagged_features(dt, maxlag=2, dropna=True):
```
Comment (fccoelho) [inline get_data.py]: Are these functions really necessary now that model training is done outside of the dashboard?

**Review by fccoelho — CHANGES_REQUESTED** [review]

File: get_data.py (hunk 3 above)
Comment (eduardocorrearaujo) [reply get_data.py]: I just removed the unnecessary functions.

**Review by fccoelho — APPROVED** [review]

**Pull request merged**